js/modules/loki_message_api.js

```diff
@@ -3,6 +3,7 @@
 /* global log, dcodeIO, window, callWorker */
 
 const fetch = require('node-fetch');
+const _ = require('lodash');
```

Comment:
They do `/* global _, ... */` instead of require('lodash') in some places in the code, what's the deal with that?
Should we be doing one way or the other or does it not really matter?

Reply:
`require` is a nodejs thing, while the /* global _, ... */ is used for front-end but it's a linter thing I think.